examples/Cliner/pipeline.py

```diff
@@ -28,13 +27,10 @@
 
 
 def do_process(input_pack_str: str):
-    # Let's assume there is a JSON string for us to use.
     datapack: DataPack = pipeline.process([input_pack_str])
-    # You can get the JSON form like this.
     data_json = datapack.serialize()
-    # Let's write it out.
     with open('generation.json', 'w') as fo:
-        fo.write(json.dumps(data_json, indent=2))
```

Comment:
It seems that those backslashes are caused by the indent parameter.

Reply:
I don't think so. This is most likely that you encode json twice. Once you have converted the object into json string, if you encode it again, the json library will escape all special characters.